src/mailroom.py

```diff
@@ -0,0 +1,122 @@
+# _*_ coding: utf-8 _*_
+import sys
+import traceback
+
+
+try:
+    input = raw_input
+except NameError:
+    pass
+
+
+DONORS = {
+    'kevin': [23, 34, 1000, 2849],
+    'bob': [234, 888, 9023],
+    'josh': [6, 1, 9, 5000, 9, 1, 2, 3],
+    'alice': [3, 1]
+}
+
+
+def interface():
+    '''Prompts user for input'''
+    response = input(u"Press 1 to send a thank you note\n"
+                     u"Press 2 to create a report\n"
+                     u"Type exit to leave the program")
```

Comment:
Notice that you have six different calls to `input` in this function?  What's the result of that?  Can you think of a way to tease these separate calls out to a single call?  What do they share in common?  What is different about each?